raiden_libs/network/matrix/client.py

```diff
@@ -401,6 +399,19 @@ def set_post_sync_hook(self, hook: Callable[[str], None]):
     def set_sync_token(self, sync_token: str) -> None:
         self.sync_token = sync_token
 
+    def set_access_token(self, user_id: str, token: str) -> None:
+        self.user_id = user_id
+        self.token = self.api.token = token
```

Comment:
Doesn't this also need to set `self.hs`? (Although it seems that attribute isn't used for anything).

Reply:
There's no need, it isn't used by anything and not even every current login/register methods set it.